docs/README.md

```diff
@@ -528,7 +528,7 @@ Represents the type of authentication used by the app. The supported options are
 
 #### `OAuthClientID`, `OAuthClientSecret` and `OAuthRedirectUri`
 
-Only valid if `AuthenticationType` is set to **OAuth**. These settings are retrieved when the developer [registers the app](https://developers.arcgis.com/applications/new).
+Only valid if `AuthenticationType` is set to **OAuth**. These settings are retrieved when the developer [registers the app](https://developers.arcgis.com/applications).
```

Comment:
I know this is very pedantic but without the trailing `/` this url creates a redirect.

```suggestion
Only valid if `AuthenticationType` is set to **OAuth**. These settings are retrieved when the developer [registers the app](https://developers.arcgis.com/applications/).
```